translations/strings.yaml

```diff
@@ -63,3 +63,22 @@ settingsDev:
   useStagingTitle: Use Staging Servers
   useStagingSubtitle: Staging services are used to test features before they go live to production
   restartRequired: VPN must be restarted to take effect
+
+splittunnel:
+  missingAppAction:
+    value:
+      - Add apps manually
+    comment:
+      - Refers to the Apps that are Missing from the Split Tunnel list
+  missingAppOne: 
+    value:
+      - \%0 Apps missing from list
+    comment:
+      - Refers to the Apps that are Missing from the Split Tunnel list
+  missingAppMultiple: 
+    value:
+      - \%0 Apps missing from list.
+    comment:
+      - Apps in the Split-Tunnel settings %1 is the amount of apps missing
+  missingAppAddedOne: Success! You added the %1 to this list.
```

Comment:
Why `%1` and not `%0`? What is `%1`? If it's the name of the app, "the NAME" doesn't make a lot of sense.